### ISSUE TITLE
Error in deployment

### ISSUE THREAD

**Issue by UseDotCom** [UseDotCom/lighthouse#398]
Created by **[usedotcom](https://github.com/usedotcom)**  at 19:30Z on 11/10/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/398))*

- - - -

Same results - only the right column displays for non-owner on Profile page. Page source of an example is posted, screenshot attached

@@@ html
<!DOCTYPE HTML><html lang="en-US"><head>
    <meta charset="UTF-8">
    <title>Dlplimited's Blog</title>
    <link rel="stylesheet" type="text/css" href="http://www.use.com/css/comment.css" />
    <link rel="stylesheet" type="text/css" href="http://www.use.com/css/use-sharing.css"/>
    <link rel="shortcut icon" href="/favicon/" type="image/png" />
    <script src="http://www.use.com/js/use-sharing.js"></script>
    
    <link rel="stylesheet" type="text/css" href="http://www.use.com/css/plugin-blog.css" />
    <link rel="stylesheet" type="text/css" href="http://www.use.com/css/smoothness/jquery-ui.css" />
    <link rel="stylesheet" type="text/css" href="http://www.use.com/js/fancybox/jquery.fancybox-1.3.4.css" />
    <script type="text/javascript">
    var blog_username = "dlplimited";
    var blog_server = "use.com";
    var moved = false;
    var copied = false;
    var showap = false;
    </script>
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js"></script>
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.13/jquery-ui.min.js"></script>
    <script type="text/javascript" src="http://www.use.com/js/fancybox/jquery.fancybox-1.3.4.js"></script>
    <script type="text/javascript" src="http://www.use.com/js/plugin-blog.js?84"></script>
    <script type="text/javascript" src="http://www.use.com/js/jquery.truncator.js"></script>
    
    <script>
        jQuery(document).ready(function(){
            jQuery("div.dontleaveme").find(".share_toolbox").mouseleave(function(){
                // God, javascript is retarded. You really can't just provide a sleep() function?
                var item = this;
                setTimeout( function() {
                    jQuery(item).hide();
                }, 1000 );
            });
            if (window.PIE) {
                $(".blog-post, #user-info").each(function() {
                    PIE.attach(this);
                });
            }

            if (moved) {
                jQuery.fancybox({
                    width : 450,
                    height : 310,
                    autoDimensions : false,
                    content : '<div style="width:auto;height:auto;overflow: auto;position:relative;"><div style="width: auto;height:auto; overflow: auto;position:relative;margin-top: 100px;text-align: center; font-family: arial, helvetica, sans-serif; color: #0000FF; font-size: 12pt;"><div style="margin: 24px 10px 0px;">Your move request has been queued for processing.</div><div style="margin: 24px 10px 0px;">Please close this window and refresh the page to check status</div></div></div>',
                    onClosed : function() {
                        document.location = 'http://dlplimited.use.com/'
                    }
                });
            } else if (copied) {
                jQuery.fancybox({
                    width : 450,
                    height : 310,
                    autoDimensions : false,
                    content : '<div style="width:auto;height:auto;overflow: auto;position:relative;"><div style="width: auto;height:auto; overflow: auto;position:relative;margin-top: 100px;text-align: center; font-family: arial, helvetica, sans-serif; color: #0000FF; font-size: 12pt;"><div style="margin: 24px 10px 0px;">Your copy request has been queued for processing.</div><div style="margin: 24px 10px 0px;">Please close this window and refresh the page to check status</div></div></div>',
                    onClosed : function() {
                        document.location = 'http://dlplimited.use.com/'
                    }
                });
            } else if (showap) {
                postToAutopostSettings()
            }
        });

        $(function() {
            $(".max150c").truncate({max_length: 150});
        });
    </script>
</head><body>
<div style="display: none; position: absolute; left: 0px; top: 0px; width: 100%; height: 200%; background-color: #fff;  opacity: .70; filter: progid:DXImageTransform.Microsoft.Alpha(opacity=70); z-index:60" id="shader"></div><div style="position: fixed;  left: 50%; top: 50%; margin-left: -204px; margin-top: -63px; width: 408px; height: 126px; display: block; visibility: visible; line-height: 4px; display: none;z-index:65" id="upload-div-floppy" >
<div style="position: absolute;line-height: 4px; left: 0px; top: 0px; width: 10px; height: 8px"><img height="8" border
="0" width="10" alt="&nbsp;" src="/images/topleft3B.gif"></div>
        <div style="position: absolute;line-height: 4px; left: 10px; top: 0px; width: 390px; height: 8px; background: url(/images/topBG3B.gif) left top repeat-x"></div>
        <div style="position: absolute;line-height: 4px; left: 398px; top: 0px; width: 10px; height: 8px"><img height="8" border="0" width="10" alt="&nbsp;" src="/images/topright3B.gif"></div>
        <div style="position: absolute;line-height: 4px; left: 0px; top: 8px; width: 10px; height: 110px; background: url(/images/sidelft3.gif) left top repeat-y"></div>
        <div style="position: absolute;line-height: 16px; left: 10px; top: 8px; width: 390px; height: 110px; background-color:#fff"> <img height="100" width="100" style="position: absolute; left: 8px; top: 5px" alt="Files Uploading..." src="/images/fl
oppywalk.gif">
        <div style="padding: 18px 0px 0px 118px"><span style="font-family: Arial, Helvetica, sans-serif; font-size: 14px">Please wait while your files are being uploaded...<br>
        &nbsp;<br>
        <div align="center"><a onclick="popOpen('/howlong.html', 450, 250)" href="#">How long will this take?</a></div>
        </span> </div>
        </div>
        <div style="line-height: 4px;position: absolute; left: 398px; top: 8px; width: 10px; height: 110px; background: url(/images/sidergt3.gif) left top repeat-y"></div>
        <div style="line-height: 4px;position: absolute; left: 0px; top: 118px; width: 10px; height: 8px"><img height="8" border="0" width="10" alt="&nbsp;" src="/images/btmleft3.gif"></div>
        <div style="line-height: 4px;position: absolute; left: 10px; top: 118px; width: 390px; height: 8px; background: url(/images/btmBG3.gif) left top repeat-x"></div>
        <div style="line-height: 4px;position: absolute; left: 398px; top: 118px; width: 10px; height: 8px"><img height="8" border="0" width="10" alt="&nbsp;" src="/images/btmright3.gif"></div>
        </div><div id="fb-root"></div>                                                                                               
<script type="text/javascript" src="/js/use-facebook.js"></script>
<script language="javascript" type="text/javascript">

</script>
    <style type="text/css">
.headerform { display: inline; margin: 0px; }
.header {width: 100%; height: 84px; }
.header td { vertical-align: middle; }
.header table { height: 80px; }
.header table td { vertical-align: middle; }
.header #userfunc, .header #userfunc a { font-family: Arial, Helvetica, sans-serif; font-size: 10pt; color: #fff; text-decoration: underline; }

#blue-top-bar {
    height: 85px;
    background-color: #3b5998;
    background: -moz-linear-gradient(left,  rgba(34,61,138,1) 0%, rgba(52,93,173,1) 16%, rgba(65,109,191,1) 21%, rgba(65,109,191,1) 29%, rgba(65,109,191,1) 35%, rgba(61,105,
186,1) 41%, rgba(52,93,173,1) 47%, rgba(34,61,138,1) 100%);
    background: -webkit-gradient(linear, left top, right top, color-stop(0%,rgba(34,61,138,1)), color-stop(16%,rgba(52,93,173,1)), color-stop(21%,rgba(65,109,191,1)), color-
stop(29%,rgba(65,109,191,1)), color-stop(35%,rgba(65,109,191,1)), color-stop(41%,rgba(61,105,186,1)), color-stop(47%,rgba(52,93,173,1)), color-stop(100%,rgba(34,61,138,1)));
    background: -webkit-linear-gradient(left,  rgba(34,61,138,1) 0%,rgba(52,93,173,1) 16%,rgba(65,109,191,1) 21%,rgba(65,109,191,1) 29%,rgba(65,109,191,1) 35%,rgba(61,105,18
6,1) 41%,rgba(52,93,173,1) 47%,rgba(34,61,138,1) 100%);
    background: -o-linear-gradient(left,  rgba(34,61,138,1) 0%,rgba(52,93,173,1) 16%,rgba(65,109,191,1) 21%,rgba(65,109,191,1) 29%,rgba(65,109,191,1) 35%,rgba(61,105,186,1)
41%,rgba(52,93,173,1) 47%,rgba(34,61,138,1) 100%);
    background: -ms-linear-gradient(left,  rgba(34,61,138,1) 0%,rgba(52,93,173,1) 16%,rgba(65,109,191,1) 21%,rgba(65,109,191,1) 29%,rgba(65,109,191,1) 35%,rgba(61,105,186,1)
 41%,rgba(52,93,173,1) 47%,rgba(34,61,138,1) 100%);
    background: linear-gradient(to right,  rgba(34,61,138,1) 0%,rgba(52,93,173,1) 16%,rgba(65,109,191,1) 21%,rgba(65,109,191,1) 29%,rgba(65,109,191,1) 35%,rgba(61,105,186,1)
 41%,rgba(52,93,173,1) 47%,rgba(34,61,138,1) 100%);
    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#223d8a', endColorstr='#223d8a',GradientType=1 );
    border-bottom: 25px solid #edf0f5;
    margin: 0 0 20px 0;
    position: relative;
    box-shadow: 0 10px 26px rgba(0, 0, 0, .2);
}
</style>
<div id="blue-top-bar">
    <table class="header" border="0" cellpadding="0" cellspacing="0">
      <tr>
        <td><div align="center">
            <table border="0" cellpadding="0" cellspacing="0">
              <tr>
                <td style="font-family: arial, helvetica, sans-serif"><a href="http://www.use.com/"><img alt="Use.com" src="/images/logo-blue.png" border="0"></a></td>
                <td style="font-family: arial, helvetica, sans-serif">&nbsp;</td>
                <td style="font-family: arial, helvetica, sans-serif"><img src="/images/tran.gif" border="0" height="1" width="10"></td>
                <td style="font-family: arial, helvetica, sans-serif"><style type="text/css">
.blue-search {
    color: #3B5998;
    border: 1px solid #999;
    border-color: #29447e #29447e #1a356e;
    font-family: 'lucida grande',tahoma,verdana,arial,sans-serif;
    -webkit-box-shadow: 0 1px 0 rgba(0, 0, 0, .1);
    font-size:11px;
    padding: 3px 6px 2px;                                                                                                                                                    
}
</style>
<form class="headerform" method="get" action="http://www.use.com/search.pl" name="sq">
    <input value="recent" name="sort" type="hidden">
    <input maxlength="1024" name="q" style="width: 240px; padding: 2px; margin-left: 10px">
    <input class="blue-search" type="submit" value="Search" style="margin-left: 8px">
</form>
</td>
                <td style="font-family: arial, helvetica, sans-serif">&nbsp;</td>
                <td id="fb-login" style="font-family: Arial, Helvetica, sans-serif"></td>
                <td style="font-family: arial, helvetica, sans-serif"><div id="control-panel">
    <script>
    var authorized = 1;
    </script>
    
<a href="javascript:doLogout('use.com');"><div class="button">Logout</div></a>

    
<a href="http://arabella7.use.com/dashboard/"><div class="button">News Feed</div></a>

    

    
<a href="http://arabella7.use.com"><div class="button">My Account</div></a>

    
<a href="http://www.use.com/pretemp.pl"><div class="button">Templates</div></a>

    

    <a href="http://www.use.com/search.pl?sort=recent"><div class="button">Most Recent</div></a>
</div>
</td>
            </table>
            </div>
    </table>
</div>
<div style="visibility:hidden" id="shade"></div><div style="visibility:hidden" id="topbox"><div style="text-align: right; width: 100%; height: 10px; margin-top: 5px"><a alt="Close" onclick="hideTop(); return false" href="#"><img height="10" border="0" width="10" src="http://www.use.com/images/closex.gif"></a><img height="10" border="0" width="5" src="http://www.'.Plugin::Blog::Config::get('domain').'/images/tran.gif"></div><iframe height="300" frameborder="0" width="420" scrolling="no" style="margin-left: 15px" marginwidth="0" marginheight="0" src="about:blank" id="innerbox"></iframe></div>
<div id="wrapper"><div id="columns"><div id="blogtitle-big"></div><div id="content">
</div><script type="text/javascript">var bg_eid='9c4c68b09b125d53ef81';</script>
<div id="bg_forced"></div>
<style type="text/css">#bg_forced { background-image : url(http://www.use.com/images/s_3/sorted/9c4c68b09b125d53ef81.jpg); background-repeat: repeat; background-position: center top;background-attachment: fixed; opacity: 1; position: fixed; left : 0px; top: 0px; width:100%; height:100%; z-index : -1000; -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=100)"; }</style><div id="user-info"><div class="avatar"><a href="/"><div class="image-div" style="text-align:center;"><img style="max-width:115px;max-height:115px;margin-top:7px;border:0;width:115px;height:115px;" src="/userthumb.pl?user=dlplimited" alt="" /></div></a></div><div class="fbtw-non" style="clear:left"><iframe src="//www.facebook.com/plugins/like.php?locale=en_US&amp;href=http%3A%2F%2Fdlplimited.use.com%2F&amp;send=false&amp;layout=button_count&amp;width=80&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;font&amp;height=21" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:80px; height:21px;" allowTransparency="true"></iframe><a href="https://twitter.com/share" class="twitter-share-button" data-url="http://dlplimited.use.com">Tweet</a><script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script></div><div class="name">dlplimited</div><div class="description max150c"></div><div class="blog-selector-title" style="margin-top: 20px;clear:both;">Share my photo blog</div><div class="share-profile" style="margin: 10px 0px 15px;"><input type="text" name="profile_url" size="20" value="http://dlplimited.use.com" onclick="this.focus(); this.select()" readonly></div><div id="share" style="height: 45px;">
            <div class="share-prof">
            <div class="share-txt3">Share</div>
            <div class="badge"><a style="border: 0px" href="#" onclick="use.sharing.facebook('http://dlplimited.use.com/', 'dlplimited\'s blog on use.com')"><img src="http://www.use.com/images/sh_fb.png"></a></div>
            <div class="badge"><a style="border: 0px" href="#" onclick="use.sharing.twitter('http://dlplimited.use.com/', 'dlplimited\'s blog on use.com')"><img src="http://www.use.com/images/sh_tw.png"></a></div>
            <div class="badge"><a style="border: 0px" href="#" onclick="use.sharing.email('http://dlplimited.use.com/', 'dlplimited\'s blog on use.com')"><img src="http://www.use.com/images/email.png"></a></div>
            <div class="badge"><a style="border: 0px" href="#" onclick="use.sharing.tumblr('http://dlplimited.use.com/', 'dlplimited\'s blog on use.com')"><img src="http://www.use.com/images/sh_tblr.png"></a></div>
            <div class="badge"><a style="border: 0px" href="#" onclick="use.sharing.reddit('http://dlplimited.use.com/', 'dlplimited\'s blog on use.com')"><img src="http://www.use.com/images/sh_redt.png"></a></div>
            <div class="badge"><a style="border: 0px" href="#" onclick="use.sharing.pinterest('http://dlplimited.use.com/', 'dlplimited\'s blog on use.com', '/userthumb.pl?user=dlplimited')"><img src="http://www.use.com/images/sh_pin.png"></a></div>
            <div class="badge"><a style="border: 0px" href="#" onclick="use.sharing.stumbleupon('http://dlplimited.use.com/', 'dlplimited\'s blog on use.com')"><img src="http://www.use.com/images/sh_su.png"></a></div>
            </div>
        </div><div id="stats">
                         <div class="stat1">
                             <ul>
                                   <li class="col1"><img src="http://www.use.com/images/prof_photo.gif" border="0"></li>
                                   <li class="col2" >475</li>
                                   <li class="col3">photos</li>
                                   <li class="col4"><img src="http://www.use.com/images/prof_video.gif" border="0"></li>
                                   <li class="col5" >0</li>
                                   <li class="col6">videos</li>
                               </ul>
                           </div>
                           <div class="stat2">
                               <ul>
                                   <li class="col7"><div><span>Country:</span><b> US </b> <img src="http://www.use.com/images/sm_flags/us.gif" /></div></li>
                                   <li class="col1"><img  src="http://www.use.com/images/prof_follow.gif" border="0"></li>
                                   <li class="col2"><a href="#" onclick="showFollowingList()">0</a></li>
                                   <li class="col8"><a href="#" onclick="showFollowingList()">followers</a></li>
                               </ul>
                           </div><br />
                     </div><div><img src="http://www.use.com/images/prof_ask.png" style="margin-left: -2px; cursor: pointer" border="0" onclick="showAskForm2('dlplimited')"></div><a style="display: block; text-decoration: none" onclick="return switchFollowing(this,1);" href="http://dlplimited.use.com"><img src="http://www.use.com/images/prof_follow.png" style="margin-left: -2px" border="0"></a><div class="qr-code"><img src="http://dlplimited.use.com/qr/?url=http%3A%2F%2Fdlplimited.use.com%2F" onclick="printPopUp()" id="qr-code" ></div><div id="copyright">&copy; 2010-2014 <span><a href="http://www.use.com/">Powered by  USE.com</a></span></div></div>
            <script>
            jQuery(document).ready(function() {
                share_links();
            });
            </script>
            <script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>

<ins class="adsbygoogle"
     style="display:inline-block;width:300px;height:250px;float:right;padding-right: 5px;"
     data-ad-client="ca-pub-8157262375494042"
     data-ad-slot="5310285025"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script></div>
<div id="clipboard"></div>
</body><html>
@@@

- - - -

Lighthouse ticket URL: http://use-com.lighthouseapp.com/projects/103799/tickets/398

Attachments:

* https://use-com.lighthouseapp.com/attachments/5575728/profile-blank.png

Was assigned to **[usedotcom](https://github.com/usedotcom)**

**Comment by UseDotCom** [comment]
By **[usedotcom](https://github.com/usedotcom)** at 19:33Z on 11/10/2014 *(via [Lighthouse](http://use-com.lighthouseapp.com/projects/103799/tickets/398))*

- - - -

As discussed, this occurs for all users when viewing someone else's Profile. Doesn't matter whether logged in or not, admin or not.

**Comment by UseDotCom** [comment]
Imported from Lighthouse  at 20:40Z on 02/05/2015

Assigned to ticket creator, **[usedotcom](https://github.com/usedotcom)**
